pages/index.tsx

```diff
@@ -33,14 +34,23 @@ const Home: NextPage = () => {
         description={t(`seo.homepage_page_description.${getCountryVariant()}`)}
         image="https://www.movapp.cz/icons/movapp-cover.jpg"
       />
-      <div className="bg-homepage-hero bg-center pt-20 pb-[10rem] pl-4 pr-4 bg-cover">
-        <h1 className="text-center max-w-3xl m-auto pt-12 pb-12 text-primary-blue text-3xl sm:text-4xl leading-snug">
-          {t(`homepage.title.${getCountryVariant()}`)}
-        </h1>
-      </div>
+      {getCountryVariant() === 'cs' || getCountryVariant() === 'sk' ? (
```

Comment:
You are using this condition three times and it's not clear the usages are connected. Consider saving its result to a meaningfully named variable and reuse that :)

`const displayYoutubeBanner = getCountryVariant() === 'cs' || getCountryVariant() === 'sk'`